### PR TITLE
Moving menu items to "Graphics Tools" directory

### PR DESCRIPTION
**Pull request by Cameron-Micka**
## Overview
Ensures all GameObject menu items are in the "Graphics Tools" directory:
image

image

## Changes
- Fixes: #82

### CODE REVIEW

File: com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/RectMask2DFastInspector.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace Microsoft.MixedReality.GraphicsTools.Editor
     [CanEditMultipleObjects]
     public class RectMask2DFastInspector : RectMask2DEditor
     {
-        [MenuItem("GameObject/UI/Rect Mask Fast - Graphics Tools")]
+        [MenuItem("GameObject/UI/Graphics Tools/Rect Mask Fast")]
```
Comment (breezinstein) [inline com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/RectMask2DFastInspector.cs:16]: Also should this be renamed Fast Rect Mask? To keep in line with the naming of the Rounded Rect Mask?

**Review by breezinstein — APPROVED** [review]

File: com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/CanvasElementMeshtInspector.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ namespace Microsoft.MixedReality.GraphicsTools.Editor
     [CustomEditor(typeof(CanvasElementMesh))]
     public class CanvasElementMeshInspector : UnityEditor.Editor
     {
-        [MenuItem("GameObject/UI/Mesh - Graphics Tools")]
+        [MenuItem("GameObject/UI/Graphics Tools/Mesh")]
```
[the remark this answers is not in the document]
Comment (Cameron-Micka) [reply com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/CanvasElementMeshtInspector.cs:14]: I could be persuaded for either. I think we made it Mesh just for brevity. What do you think is better?

File: com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/RectMask2DFastInspector.cs (hunk 1 above)
Comment (Cameron-Micka) [reply com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/RectMask2DFastInspector.cs:16]: Good idea!

File: com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/CanvasElementMeshtInspector.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Cameron-Micka) [reply com.microsoft.mrtk.graphicstools.unity/Editor/Inspectors/CanvasElementMeshtInspector.cs:14]: Going to merge for now, we can change this in another PR if need be!

**Pull request merged**